systemd/ceph-exporter@.service.in

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Ceph cluster exporter daemon
+PartOf=ceph-exporter.target
+After=network-online.target local-fs.target time-sync.target
+Before=remote-fs-pre.target ceph-exporter.target
+Wants=network-online.target local-fs.target time-sync.target remote-fs-pre.target ceph-exporter.target
+
+[Service]
+Environment=CLUSTER=ceph
```

Comment:
what is this for? i searched the tree for the place(s) where we `getenv("CLUSTER")`, not nothing stands out.

i guess you were replicating the pattern of existing service units. but i don't think this line still applies even in them.

Reply:
ahh, thank you for pointing this out. sorry, i missed it.